ResearchKit/Common/ORKHelpers.m

```diff
@@ -395,6 +395,84 @@ CGFloat ORKTableViewLeftMargin(UITableView *tableView) {
     return font;
 }
 
+NSURL *ORKURLFromBookmarkData(NSData *data) {
+    if (data == nil) {
+        return nil;
+    }
+    
+    BOOL bookmarkIsStale = NO;
+    NSError *bookmarkError = nil;
+    NSURL *bookmarkURL = [NSURL URLByResolvingBookmarkData:data
+                                                   options:NSURLBookmarkResolutionWithoutUI
+                                             relativeToURL:nil
+                                       bookmarkDataIsStale:&bookmarkIsStale
+                                                     error:&bookmarkError];
+    if (! bookmarkURL) {
+        ORK_Log_Debug(@"Error loading URL from bookmark: %@", bookmarkError);
```

Comment:
You're probably right. Though really we just need to give framework users an easy way to turn it off rather than hijacking DEBUG. I'm not sure there's a real need for more than two levels; but there needs to be a way to turn some or all of it off without affecting your own app's console logging.

I am a bit cautious about the use of Oops, which unconditionally logs. The issue is that anything going to the console is inherently public. If the log message were to leak information about your health, that's generally not a good thing.

In any event, not a change that's going to happen in this PR. Filed #256.